pkg/operator/controller/ingress/controller.go

```diff
@@ -773,13 +774,80 @@ func (r *reconciler) validate(ic *operatorv1.IngressController) error {
 	if err := validateClientTLS(ic); err != nil {
 		errors = append(errors, err)
 	}
+	if err := validateHTTPHeaders(ic); err != nil {
```

Comment:
Overall, I'm not sure we need to reject admission if the header editing has formatting issues.  The controller can still function properly, so why would we want to reject it?

Reply:
we want to reject because invalid headers cause haproxy to fail

Reply:
file no longer present for this PR